core/src/main/java/hudson/FilePath.java

```diff
@@ -1321,10 +1321,13 @@ public String getName() {
     }
 
     /**
-     * Short for {@code getParent().child(rel)}. Useful for getting other files in the same directory. 
+     * Short for {@code getParent().child(rel)}. Useful for getting other files in the same directory.
+     * @return null if {@link #getParent} would have
```

Comment:
https://github.com/jenkinsci/jenkins/pull/4527#discussion_r383917150 is IMHO better but 🤷‍♂ as long as we have user docs